oracledb-mixin/dashboards/overview.libsonnet

```diff
@@ -1113,7 +1113,7 @@ local tablespaceSizePanel = {
               template.new(
                 'tablespace',
                 promDatasource,
-                query='label_values(oracledb_tablespace_bytes{' + matcher + '}, tablespace)',
+                query='label_values(oracledb_tablespace_bytes{%(oracledbSelector)s}, tablespace)' % $._config,
```

Comment:
any reason why this didn't stay as "matcher"?

Reply:
I replaced the static `matcher` variable with the `getMatcher` function. Not sure if function is the appropriate term. 

With `matcher` not existing anymore, this looks at the config and pulls selectors from there, to my knowledge. The portions that still contain `' + matcher + '` are going to be the panels that take that in as an argument.

In the `addPanels` section, the `getMatcher` function takes in the config and returns the appropriate selectors.
```
databaseStatusPanel(getMatcher($._config))
```

Reply:
@algchoo sure, I get that. But why did this query switch from using `matcher` to `oracledbSelector`?

Reply:
@StefanKurek The variable `matcher` no longer exists, so we have to look elsewhere for the selectors. I'm not sure I could use the `getMatcher` function and this is the [example](https://github.com/grafana/jsonnet-libs/pull/1158/files) I have been referring to. Is there a different approach I might take here?

Reply:
Also, just noticed that this replacement nerfs including `instance` as part of the selectors, added that back.

Reply:
@algchoo ok sorry. Was missing that matcher wasn't being passed in here.